aioauth/oidc/core/requests.py

```diff
@@ -1,19 +1,31 @@
 from dataclasses import dataclass, field
 
-from typing import Any, Optional
+from typing import Any, Optional, TypeVar
 
 from aioauth.requests import BaseRequest, Query as BaseQuery, Post
 
 
 @dataclass
 class Query(BaseQuery):
+    # Space delimited, case sensitive list of ASCII string values that
+    # specifies whether the Authorization Server prompts the End-User for
+    # reauthentication and consent. The defined values are: none, login,
+    # consent, select_account.
+    # https://openid.net/specs/openid-connect-core-1_0.html#AuthRequest
     prompt: Optional[str] = None
```

Comment:
How about turning this comment into documentation for the `prompt` attribute?

```suggestion
    prompt: Optional[str] = None
    """
    Space delimited, case sensitive list of ASCII string values that
    specifies whether the Authorization Server prompts the End-User for
    reauthentication and consent. The defined values are: none, login,
    consent, select_account.
    https://openid.net/specs/openid-connect-core-1_0.html#AuthRequest
    """
```